packages/plugin-pyth-data/src/utils/publisherCapsValidation.ts

```
@@ -0,0 +1,120 @@
+ */
+export function validatePublisherAddress(publisher: string): boolean {
+    return /^0x[a-fA-F0-9]{40}$/.test(publisher);
+}
+
+/**
+ * Validates a cap value
+ * @param cap Cap value to validate
+ * @returns boolean True if cap is valid
+ */
+export function validateCap(cap: number): boolean {
+    return !isNaN(cap) && cap >= 0;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)